Load/ontology/General/collections/popbio.yaml

```diff
@@ -0,0 +1,154 @@
+stable_id: POPBIO_8000100
+member: inversion
+memberPlural: inversions
+isProportion: false
+isCompositional: false
+---
+label: prevalence of variants at protein locus AChE1 G119
+stable_id: POPBIO_8000086
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus AChE1 G280
+stable_id: POPBIO_8000101
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus AChE1 G280
+stable_id: POPBIO_8000107
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Gste2 I114
+stable_id: POPBIO_8000104
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Gste2 L119
+stable_id: POPBIO_8000097
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr D1763
+stable_id: POPBIO_8000088
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr F1534
+stable_id: POPBIO_8000089
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr I1011
+stable_id: POPBIO_8000090
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr I1532
+stable_id: POPBIO_8000091
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr L1014
+stable_id: POPBIO_8000092
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr L995
+stable_id: POPBIO_8000109
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr N1575
+stable_id: POPBIO_8000093
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr S989
+stable_id: POPBIO_8000094
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Kdr V1016
+stable_id: POPBIO_8000095
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label: prevalence of variants at protein locus Rdl A296
+stable_id: POPBIO_8000099
+member: allele
+memberPlural: alleles
+isProportion: true
+isCompositional: true
+---
+label:  presence/absence of arboviruses
+stable_id: POPBIO_8000030
+member: virus
+memberPlural: viruses
+isProportion: false
+isCompositional: false
+---
+label:  prevalence of arboviruses
+stable_id: POPBIO_8000036
+member: virus
+memberPlural: viruses
+isProportion: false
```

Comment:
so some prevalence collections are proportions and some are not? That's totally fine just verifying!

Reply:
I thought maybe it should be a proportion, but Danielle's advice in your PR was to say 'false'. Maybe it was a typo.

tbh I was wondering...